scripts/fast-reboot

```diff
@@ -13,6 +13,9 @@ STRICT=no
 REBOOT_METHOD="/sbin/kexec -e"
 ASSISTANT_IP_LIST=""
 ASSISTANT_SCRIPT="/usr/bin/neighbor_advertiser"
+DEVPATH="/usr/share/sonic/device"
+PLATFORM=$(sonic-cfggen -H -v DEVICE_METADATA.localhost.platform)
+PLATFORM_PLUGIN="fastboot_plugin"
```

Comment:
You can change the PLUGIN name to "${REBOOT_TYPE}_plugin" so that warm reboot and fast reboot could have different plugins.

Reply:
what is the reason to have fastboot_plugin per platform?